wordpress-mysql-replication/azuredeploy.json

```diff
@@ -369,38 +363,15 @@
                     "workerSize": {
                         "value": "[parameters('workerSize')]"
                     },
-                    "webDeploy_AuthenticationKey": {
-                        "value": "[variables('wordpressAuthKeys')]"
-                    },
-                    "webDeploy_SecureAuthenticationKey": {
-                        "value": "[variables('wordpressAuthKeys')]"
-                    },
-                    "webDeploy_LoggedInKey": {
-                        "value": "[variables('wordpressAuthKeys')]"
-                    },
-                    "webDeploy_NonceKey": {
-                        "value": "[variables('wordpressAuthKeys')]"
-                    },
-                    "webDeploy_AuthenticationSalt": {
-                        "value": "[variables('wordpressAuthKeys')]"
-                    },
-                    "webDeploy_SecureAuthenticationSalt": {
-                        "value": "[variables('wordpressAuthKeys')]"
-                    },
-                    "webDeploy_LoggedInSalt": {
-                        "value": "[variables('wordpressAuthKeys')]"
-                    },
-                    "webDeploy_NonceSalt": {
-                        "value": "[variables('wordpressAuthKeys')]"
-                    },
+
                     "DbServer": {
-                        "value": "[concat(parameters('dnsName'), '.', resourceGroup().location, '.cloudapp.azure.com')]"
+                        "value": "[concat(parameters('dnsName'), '.', resourceGroup().location, '.cloudapp.azure.com:',parameters('mysqlFrontEndPort0'))]"
```

Comment:
This is specifically for Public Azure. For mooncake , Azure stack it depends if the resources ( with the version mentioned in the template ) in the json template are supported .